src/temp/ProjectMapTabs/Tabs.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styles from './ProjectMapTabs.module.scss';
+import SingleTab from './SingleTab';
+import SatelliteAnalysisIcon from '../icons/SatelliteAnalysisIcon';
+import { useTranslation } from 'next-i18next';
+import FieldDataIcon from '../icons/FieldDataIcon';
+import SatelliteIcon from '../../../public/assets/images/icons/SatelliteIcon';
+
+interface TabsProps {
+  selected: string;
+}
+
+const Tabs = ({ selected }: TabsProps) => {
+  const [selectedMode, setSelectedMode] = useState(selected);
+
+  const allTabsList = ['satellite', 'field', 'timeTravel'];
+  const setSeparatorVisibility = (
+    selectedMode: string,
+    separatorId: number
+  ) => {
+    const index = allTabsList.indexOf(selectedMode);
+    if (separatorId !== index - 1 && separatorId !== index) return true;
+    return false;
+  };
+
+  const { t } = useTranslation(['maps', 'projectDetails']);
+  return (
+    <div className={styles.tabsContainer}>
+      <div onClick={() => setSelectedMode('satellite')}>
+        <SingleTab
+          icon={
+            <SatelliteAnalysisIcon
+              color={
+                selectedMode === 'satellite'
+                  ? `${'var(--light)'}`
+                  : `${'var(--dark)'}`
+              }
+            />
+          }
+          title={t('projectDetails:satelliteAnalysis')}
+          isSelected={selectedMode === 'satellite'}
+        />
```

Comment:
How about removing the outer div, and supplying the onClick handler to the SingleTab component, where it can be triggered directly by the `button` element?